tests/marketplace/deployer_helm_tiller_base/onbuild/standard/run_test

```diff
@@ -22,6 +22,9 @@ docker run \
   > dev
 chmod a+x dev
 
+# Runs diagnostic tool to make sure your environment is properly setup.
```

Comment:
Can you do this for the other tests as well please? All of them are under `/tests/marketplace/`

Reply:
Done.